### ISSUE TITLE
More options button on a comment should have a bigger tap area

### ISSUE THREAD

**Issue by BasThomas** [GitHawkApp/GitHawk#2006]
![GitHawk Upload by BasThomas](https://i.imgur.com/gerlDJ9.jpg)

<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.22.0 (1532907001)
Device: iPhone X (iOS 11.4.1)
TestFlight: true
</pre>
</details>
